<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StreamWatcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="fileExplorerButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSelectedFileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add File</string>
    </property>
   </widget>
   <widget class="QListWidget" name="tempListWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addFilesToWatchListButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Files to Watch List</string>
    </property>
   </widget>
   <widget class="QListWidget" name="watchedFilesList">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editOutPutDirButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="staticOutputDirLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Dir:</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputDirLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
